fusor-ember-cli/app/components/tr-subscription.js

```diff
@@ -36,25 +36,36 @@ export default Ember.Component.extend({
     return (this.get('subscription.qtyAvailable') === 0);
   }),
 
-  setDefaultQtyToAttach: Ember.on('didInsertElement', function() {
```

Comment:
removing `setDefaultQtyToAttach` since it caused problems and is not needed. Please it's not accurate as it was done when my thought progress was these was the total subscriptions for the deployment and not just added ones.